packages/AlgernonTests.package/ALGTestTypes.class/instance/testWindowCollapse.st

```diff
@@ -0,0 +1,10 @@
+testing window type
+testWindowCollapse
+
+	|browser|
+	
+	browser := self openBrowser.
+	self testOpenWindowEntry: 'System Browser' withAction: 'collapse'.
+	Transcript show: browser isCollapsed.
```

Comment:
Is this Transcript printing still intended or was it just for debugging? ^^

Reply:
100% intended :D